docs/src/main/asciidoc/security-built-in-authentication-support-concept.adoc

```diff
@@ -3,15 +3,17 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
+[id="security-built-in-authentication-support-concept"]
 = Built-In Authentication Support
-
 include::_attributes.adoc[]
+:categories: security,web
 
 The following section describes the Quarkus built-in authentication mechanisms for HTTP based FORM, BASIC, and Mutual TLS authentication. Proactive authentication is also described.
 
-include::security-basic-auth-concept.adoc[leveloffset=+1]
+include::security-basic-authentication-concept.adoc[leveloffset=+1]
 
-include::security-enabling-basic-auth-howto.adoc[leveloffset=+2]
+// include::security-enabling-basic-auth-howto.adoc[leveloffset=+2]
+// How to I convert this how-to into a guide that appears alongside all the other guides?
```

Comment:
How do I convert this how-to into a guide that appears alongside all the other guides?

Reply:
@rolfedh Hi, can you check with Michelle please as she has opened #29418, please also see #29472 just in case

Reply:
@rolfedh -  As part of the Diataxis restructuring for the Basic authentication content in the Security section, I have renamed this file to security-basic-authentication-howto.adoc, plus there is some discussion with @sberyozkin about this content that could change requirements around it. 

See PR https://github.com/quarkusio/quarkus/pull/29418 

When the content filename, ID, and categories are correctly populated and merged, the Quarkus Index page gets rebuilt from this repo here: https://github.com/quarkusio/quarkusio.github.io

Reply:
My bad. I didn't realize this MR would get merged in my absence. I intended to discuss it with other members of the team before continuing. I will put others like it on hold in the future.